### DISCUSSION TITLE
Adoption of ZXing.Net.Maui into the Community Toolkit

### DISCUSSION THREAD

**Discussion by baaaaif** [CommunityToolkit/Maui#1090] [New Feature Discussions]
Since @Redth seems to be very busy with MAUI itself, could you talk to him about including ZXing.Net.Maui in the Community Toolkit here?
It's pretty much the standard library when it comes to scanning barcodes. However, there is no recent progress and there is a lack of support for Windows (however, there is a pull request from @paoldev (https://github.com/Redth/ZXing.Net.Maui/pull/83)

**Comment by VladislavAntonyuk** [comment]
I would say it is rather big to be migrated and included in MCT.

**Comment by pictos** [comment]
@baaaaif from an old conversation with Redth, the solution will be another way around. That's, ZXing.Net.Maui take a dependency on MCT's CameraView. It will be better to keep it in a separate project, from the time of that conversation, reuse the CameraView implementation, and just focus on the scan-bar feature.

I'll double-check if the plan remains, but I think so

**Reply by pictos** [reply]
The Windows support is because WinUI doesn't have a `CameraView` control implemented, so if you need I suggest to open an issue at WinUI repo

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
